src/core/binformat.js

```diff
@@ -116,7 +120,8 @@ const FIELDS_MAP = exports.fields = {
     17: 'InvoiceID',
     18: 'Nickname',
     19: 'Amendment',
-    20: 'TicketID'
+    20: 'TicketID',
```

Comment:
@sublimator As far as I know Tickets are in suspended animation.  When we revisit out-of-order transactions we'll see whether Tickets are the right approach or if there's a better way.